packages/nimble-components/src/table-column/anchor/index.ts

```diff
@@ -0,0 +1,158 @@
+/* eslint-disable max-classes-per-file */
+import { DesignSystem } from '@microsoft/fast-foundation';
+import { attr } from '@microsoft/fast-element';
+import { TableColumn } from '../base';
+import { styles } from '../base/styles';
+import { template } from '../base/template';
+import { TableColumnSortOperation } from '../base/types';
+import { mixinFractionalWidthColumnAPI } from '../mixins/fractional-width-column';
+import { mixinGroupableColumnAPI } from '../mixins/groupable-column';
+import type { TableStringField } from '../../table/types';
+import { tableColumnAnchorCellViewTag } from './cell-view';
+import { tableColumnTextGroupHeaderTag } from '../text/group-header-view';
+import type { AnchorAppearance } from '../../anchor/types';
+
+export type TableColumnAnchorCellRecord = TableStringField<'label' | 'href'>;
+export interface TableColumnAnchorColumnConfig {
+    placeholder: string;
+    appearance: AnchorAppearance;
+    hreflang?: string;
+    ping?: string;
+    referrerpolicy?: string;
+    rel?: string;
+    target?: string;
+    type?: string;
+    download?: string;
+}
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-table-column-anchor': TableColumnAnchor;
+    }
+}
+
+/**
+ * A table column for displaying links.
+ */
+export class TableColumnAnchor extends mixinGroupableColumnAPI(
+    mixinFractionalWidthColumnAPI(TableColumn<TableColumnAnchorColumnConfig>)
+) {
+    @attr({ attribute: 'label-field-name' })
+    public labelFieldName?: string;
+
+    @attr({ attribute: 'href-field-name' })
+    public hrefFieldName?: string;
+
+    @attr
+    public placeholder?: string;
+
+    @attr
+    public appearance?: AnchorAppearance;
+
+    @attr
+    public hreflang?: string;
+
+    @attr
+    public ping?: string;
+
+    @attr
+    public referrerpolicy?: string;
+
+    @attr
+    public rel?: string;
+
+    @attr
+    public target?: string;
+
+    @attr
+    public type?: string;
+
+    @attr
+    public download?: string;
+
+    public constructor() {
+        super({
+            cellRecordFieldNames: ['label', 'href'],
+            cellViewTag: tableColumnAnchorCellViewTag,
+            groupHeaderViewTag: tableColumnTextGroupHeaderTag,
+            delegatedEvents: ['click']
+        });
+        this.columnInternals.sortOperation = TableColumnSortOperation.localeAwareCaseSensitive;
```

Comment:
Do we have any tests that verify sorting works correctly for the anchor column? They don't need to be as comprehensive as our general sorting tests, but at least one test which verifies that sorting works and sorts by the display text and not the href seems like a good idea.

Reply:
Added a test to verify we sort by the right field.

Reply:
Added one test that verifies we sort by the label and not the href.